data/combine-requests/combine-requests.js

```diff
@@ -245,12 +246,18 @@ module.exports = connect.behavior("data/combine-requests",function(base){
 					combineDataPromise.then(function(combinedData){
 						// farm out requests
 						forEach.call(combinedData, function(combined){
+							// clone combine.set to prevent mutations by base.getListData
+							var combinedSet = deepAssign({}, combined.set);
 
-							base.getListData(combined.set).then(function(data){
+							base.getListData(combinedSet).then(function(data){
 								if(combined.pendingRequests.length === 1) {
 									combined.pendingRequests[0].deferred.resolve(data);
 								} else {
 									forEach.call(combined.pendingRequests, function(pending){
+										// get the subset using the combine.set property before being passed down
```

Comment:
maybe change `combine.set` to `combineSet` to keep with the variable name changing.